base/install-functions.sh

```diff
@@ -1,3 +1,11 @@
+get_module_path() {
+    for module; do
+        if [ -n "$module" ]; then
+            grep -E "^$module=" $EPICS_RELEASE_FILE
+        fi
```

Comment:
I'm not sure if that's the best way to do it. I don't want to restrict the parameter to have only lines with content. This would require the modules to be all in the same line.

Let me know if you know way better way to handle this type of parameter.

Reply:
This works and doesn't seem overly inefficient, so I don't see any issue with it.

What do you think about supporting manually specified dependencies anyway (check if there's a `=`, if yes, just pass through the whole line)? Some weird case might require it (but then it could also be more correct to fix epics-in-docker to include that specific dep in the release file).

Reply:
> This works and doesn't seem overly inefficient, so I don't see any issue with it.

Great! The previous version of it was worse and buggy, nice to have it right at last.

> What do you think about supporting manually specified dependencies anyway (check if there's a =, if yes, just pass through the whole line)?

I had thought about that, but in a different way. I would change the install function to conditionally overwrite configure/RELEASE before installing based on the emptiness of the module list. But I dropped the idea when I thought to myself whether we wanted or not to enforce all installed modules to be listed there.

> Some weird case might require it

Maybe a module that requires (a submodule of) itself? Not sure how wild module dependencies can become.

Reply:
> Maybe a module that requires (a submodule of) itself? Not sure how wild module dependencies can become.

I dearly hope that's not the case... I think we can omit it for now.